### ISSUE TITLE
retrain_yolo.py almost have error code

### ISSUE THREAD

**Issue by cxf2015** [allanzelener/YAD2K#78]
function process_data():
orig_size = np.array([[image.width, image.height] for image in (images)])
and
boxes_xy = [boxxy / orig_size[i] for i, boxxy in enumerate(boxes_xy)]
        boxes_wh = [boxwh / orig_size[i] for i, boxwh in enumerate(boxes_wh)]
is that right?

**Comment by Firyuza** [comment]
@cxf2015  Yes, it's right. We need to convert box coordinates to interval [0,1]. Because the ground truth coordinates are presented relative the image size. But when you train you network or in inference stage we work with [0,1] values.

**Comment by y-kl8** [comment]
Hi, do you know how to convert the VOC data into .npz format, when I want to run retrain_yolo.py? @cxf2015
